packages/kolibri-tools/package.json

```diff
@@ -22,6 +22,7 @@
     "babel-preset-env": "^1.7.0",
     "buble": "0.19.3",
     "buble-loader": "0.5.0",
+    "check-node-version": "^3.3.0",
```

Comment:
Can probably remove `engine-strict` below, as it seems this is taking its place?

Reply:
Ah yes, I totally I removed it, but I think I reverted the change when I was trying different replacements.